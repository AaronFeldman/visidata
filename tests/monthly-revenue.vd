sheet	col	row	keystrokes	input	comment
	sheet	0	o	sample_data/sample.tsv	open input in VisiData
sample	OrderDate	0	;	(\d+-\d+)	add new column from capture groups of regex; also requires example row
sample	OrderDate	0	-		hide current column (to unhide go to Columns sheet and edit its width)
sample	OrderDate_re0	0	^	OrderDate	edit name of current column
sample	Total	0	$		set type of current column to currency
sample	Total	0	+	sum	add aggregator to the current column
sample	OrderDate	0	F		open Frequency Table grouped on current column
sample_OrderDate_freq	OrderDate	0	[		sort ascending by current column
sample_OrderDate_freq	sum_Total	0	^	Revenue	edit name of current column
