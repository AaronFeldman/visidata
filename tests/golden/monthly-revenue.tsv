OrderDate	count	Revenue
2016-01	2	1,188.55
2016-02	2	719.37
2016-03	1	167.44
2016-04	2	448.65
2016-05	2	512.78
2016-06	2	988.50
2016-07	2	1,676.90
2016-08	1	174.65
2016-09	2	505.84
2016-10	2	827.08
2016-11	2	778.89
2016-12	2	1,269.69
2017-01	1	413.54
2017-02	2	1,324.96
2017-03	2	389.43
2017-04	2	610.38
2017-05	2	787.57
2017-06	1	625.00
2017-07	2	996.33
2017-08	2	1,830.90
2017-09	2	160.27
2017-10	2	1,157.49
2017-11	1	54.89
2017-12	2	2,018.78
